charts/rancher-logging/rancher-logging/100.0.1+up3.15.0/templates/loggings/kube-audit/logging.yaml

```diff
@@ -1,113 +1,24 @@
-      {{- if .Values.fluentbit.inputTail.Skip_Long_Lines }}
-      Skip_Long_Lines: {{ .Values.fluentbit.inputTail.Skip_Long_Lines | toString | quote }}
-      {{- end }}
-    extraVolumeMounts:
-    - source: {{ template "kubeAuditPathPrefix" . }}
-      destination: "/kube-audit-logs"
-      readOnly: true
-    {{- if not .Values.disablePvc }}
-    {{- with .Values.fluentbit.bufferStorage }}
-    bufferStorage: {{- toYaml . | nindent 6 }}
-    {{- end }}
-    {{- with .Values.fluentbit.bufferStorageVolume }}
-    bufferStorageVolume: {{- toYaml . | nindent 6 }}
-    {{- end }}
-    {{- end }}
-    {{- if .Values.global.psp.enabled }}
-    security:
-      podSecurityPolicyCreate: true
-      roleBasedAccessControlCreate: true
-    {{- end }}
-    {{- with (concat (.Values.tolerations) (.Values.additionalLoggingSources.kubeAudit.fluentbit.tolerations)) }}
-    tolerations: 
-      {{- toYaml . | nindent 6 }}
-    {{- end }}
-    nodeSelector:
-      {{- include "controlplaneSelector" . | nindent 6 }}
-    {{- with .Values.fluentbit.resources }}
-    resources: {{- toYaml . | nindent 6 }}
-    {{- end }}
-    {{- if (include "requireFilterKubernetes" .) }}
-    filterKubernetes:
-      {{- if .Values.fluentbit.filterKubernetes.Merge_Log }}
-      Merge_Log: "{{ .Values.fluentbit.filterKubernetes.Merge_Log }}"
-      {{- end }}
-      {{- if .Values.fluentbit.filterKubernetes.Merge_Log_Key }}
-      Merge_Log_Key: "{{ .Values.fluentbit.filterKubernetes.Merge_Log_Key }}"
-      {{- end }}
-      {{- if .Values.fluentbit.filterKubernetes.Merge_Log_Trim }}
-      Merge_Log_Trim: "{{ .Values.fluentbit.filterKubernetes.Merge_Log_Trim }}"
-      {{- end }}
-      {{- if .Values.fluentbit.filterKubernetes.Merge_Parser }}
-      Merge_Parser: "{{ .Values.fluentbit.filterKubernetes.Merge_Parser }}"
-      {{- end }}
-    {{- end }}
-  fluentd:
-    image:
-      repository: {{ template "system_default_registry" . }}{{ .Values.images.fluentd.repository }}
-      tag: {{ .Values.images.fluentd.tag }}
-    configReloaderImage:
-      repository: {{ template "system_default_registry" . }}{{ .Values.images.config_reloader.repository }}
-      tag: {{ .Values.images.config_reloader.tag }}
-    {{- if not .Values.disablePvc }}
-    {{- with .Values.fluentd.bufferStorageVolume }}
-    bufferStorageVolume: {{- toYaml . | nindent 6 }}
-    {{- end }}
-    {{- end }}
-    disablePvc: {{ .Values.disablePvc }}
-    {{- if .Values.fluentd.replicas }}
-    scaling:
-      replicas: {{ .Values.fluentd.replicas }}
-    {{- end }}
-    {{- if .Values.global.psp.enabled }}
-    security:
-      podSecurityPolicyCreate: true
-      roleBasedAccessControlCreate: true
-    {{- end }}
-    {{- with (default .Values.tolerations .Values.fluentd.tolerations) }}
+    {{- with (concat (.Values.tolerations) (.Values.fluentbit.tolerations) (.Values.additionalLoggingSources.kubeAudit.fluentbit.tolerations)) }}
     tolerations: {{- toYaml . | nindent 6 }}
     {{- end }}
```

Comment:
Why do these tolerations merge differently than the cloud provider and k3s CRs?

Reply:
Not sure if there was a specific reason originally but in order to not introduce any breaking changes I kept it the same.